test/cases/phq9/phq9_question10_case.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Ph1-9 Question 10 Score
+externalData:
+- phq9_reusable_resources
+data:
+-
+  *patientResource
+-
+  <<: *questionnaireResource
+  item:
+  -
+    *Phq9Question10
```

Comment:
This YAML is formatted _very_ oddly, we should probably run it through `yamllint` at some point (out of scope for this PR). It may not seem like a big deal, but improving the whitespace could help a lot with readability and prevent errors